microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/graphql/properties/MicroProfileGraphQLProvider.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+* Copyright (c) 2020 IBM Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     IBM Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.lsp4mp.jdt.internal.graphql.properties;
+
+import org.eclipse.lsp4mp.jdt.internal.graphql.MicroProfileGraphQLConstants;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.lsp4mp.jdt.core.AbstractStaticPropertiesProvider;
+import org.eclipse.lsp4mp.jdt.core.MicroProfileCorePlugin;
+import org.eclipse.lsp4mp.jdt.core.SearchContext;
+import org.eclipse.lsp4mp.jdt.core.utils.JDTTypeUtils;
+
+/**
+ * Properties provider to collect MicroProfile properties from MicroProfile GraphQL 
+ * error handling
+ * 
+ * @author Kathryn Kodama
+ * 
+ * @see https://github.com/eclipse/microprofile-graphql/blob/905456693d9bfb1f78efe0d7e614db6fde3324b2/server/spec/src/main/asciidoc/errorhandling.asciidoc
+ *
+ */
+public class MicroProfileGraphQLProvider extends AbstractStaticPropertiesProvider {
+
+    public MicroProfileGraphQLProvider() {
+        super(MicroProfileCorePlugin.PLUGIN_ID, "/static-properties/mp-graphql-metadata.json");
+    }
+
+    @Override
+    protected boolean isAdaptedFor(SearchContext context, IProgressMonitor monitor) {
+		// Check if MicroProfile GraphQL exists in classpath
+		IJavaProject javaProject = context.getJavaProject();
+		return (JDTTypeUtils.findType(javaProject, MicroProfileGraphQLConstants.GRAPHQL_NAME_TAG) != null);
+	}
```

Comment:
Something seems off with the indentation here. I think `super` and the `IJavaProject` should line up. Same w/ the braces

<img src="https://user-images.githubusercontent.com/16768710/89666139-d69d6a00-d8a7-11ea-89b9-e4bd3df47e7d.png" width=350>